### PR TITLE
Updates to Integration List

### PR DESCRIPTION
**Pull request by kahboom**
- Increase width of icon column.
- Add toggles for state.
- Cleanup.
- Fix bug.
- Ah my brain is so fried that I can't remember what I did.

### CODE REVIEW

File: src/app/integrations/list/list.component.ts [hunk 1]
```diff
@@ -17,6 +17,7 @@ import { log, getCategory } from '../../logging';
 export class IntegrationsListComponent {
 
   private toast;
+  isActive = true;
```
Comment (gashcrumb) [inline src/app/integrations/list/list.component.ts]: What's this for?

File: src/app/integrations/list/list.component.html [hunk 2]
```diff
@@ -106,13 +106,16 @@ <h4 class="modal-title pull-left">Confirm {{ getActionTitle() }}</h4>
               </h3>
             </div>
 
-            <div *ngIf="integration.currentStatus === 'Activated' || integration.currentStatus === 'Deactivated'">
+            <div *ngIf="integration.currentStatus === 'Activated' || integration.currentStatus === 'Deactivated'"
+                 (click)="toggleActive(integration)">
```
Comment (gashcrumb) [inline src/app/integrations/list/list.component.html]: +1

File: src/app/integrations/list/list.component.ts (hunk 1 above)
Comment (kahboom) [inline src/app/integrations/list/list.component.ts]: We can remove that now, let me do that.

**Comment by gashcrumb** [comment]
oh lint:

```
/home/ubuntu/ipaas-ui/src/app/integrations/list/list.component.ts[148, 7]: missing whitespace
```

**Pull request merged**